backend/src/main/java/de/neuefische/service/TankService.java

```diff
@@ -13,11 +13,15 @@ public class TankService {
 
     private final TankRepo tankRepo;
 
-    public List<Tank> getAllTanks(){
+    public List<Tank> getAllTanks() {
         return tankRepo.findAll();
     }
 
     public Tank addTank(Tank newTank) {
         return tankRepo.save(newTank);
     }
+
+    public void deleteTank(String id) {
+        tankRepo.deleteById(id);
```

Comment:
public String methode with return e.g. "tank successfully deleted" should be better